doc/source/user/troubleshooting-importerror.rst

```diff
@@ -133,10 +133,17 @@ your system. If you can open a correct python shell, you can also run the
 following in python::
 
     import os
-    PYTHONPATH = os.environ['PYTHONPATH'].split(os.pathsep)
-    print("The PYTHONPATH is:", PYTHONPATH)
-    PATH = os.environ['PATH'].split(os.pathsep)
-    print("The PATH is:", PATH)
+    PYTHONPATH = os.environ.get('PYTHONPATH', None)
+    if PYTHONPATH is None:
+        print("The PYTHONPATH is not used.")
+    else:
+        print("The PYTHONPATH includes:", PYTHONPATH.split(os.pathsep))
+
+    PATH = os.environ.get('PATH', None)
+    if PATH is None:
+        print("The PATH is not used.")
+    else:
+        print("The PATH includes:", PATH.split(os.pathsep))
```

Comment:
```suggestion
    print("PYTHONPATH: ", os.environ.get('PYTHONPATH'))
    print("PATH: ", os.environ.get('PATH'))
```
A slightly shorter suggestion without the nice "{} is not used" message. Alternatively, we could suggestion running these in an interpreter, though I suppose having it written out in this form is nicer for those who might want to run it as a script to introspect the setup of Python embedded in an IDE etc.

Reply:
Yeah, if I don't mind `['']` being printed for an empty path (which I guess is fine), can even add the splitting to make it easier for some users to read...